### PR TITLE
[WIP] mgencur: Add eventings 0.23 2nd try extras

### PR DESCRIPTION
**Pull request by mgencur**
Trying with increased logging.

### CODE REVIEW

File: hack/lib/serverless.bash [hunk 1]
```diff
@@ -218,7 +218,14 @@ metadata:
...
 spec:
-  {}
+  config:
+    logging:
+      loglevel.controller: DEBUG
+      loglevel.webhook: DEBUG
+      loglevel.kafkachannel-dispatcher: DEBUG
+      loglevel.kafkachannel-controller: DEBUG
+      loglevel.inmemorychannel-dispatcher: DEBUG
+      loglevel.mt-broker-controller: DEBUG
```
Comment (cardil) [inline hack/lib/serverless.bash:228]:
```suggestion
  config:
    logging:
      loglevel.controller: DEBUG
      loglevel.webhook: DEBUG
      loglevel.kafkachannel-dispatcher: DEBUG
      loglevel.kafkachannel-controller: DEBUG
      loglevel.inmemorychannel-dispatcher: DEBUG
      loglevel.mt-broker-controller: DEBUG
      loglevel.kafka-controller: DEBUG
      loglevel.kafkasource: DEBUG
      loglevel.kafkachannel-webhook: DEBUG
```

**Comment by cardil** [comment]
In [build 1418188685278449664](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift-knative_serverless-operator/1115/pull-ci-openshift-knative-serverless-operator-main-4.7-upgrade-tests-aws-ocp-47/1418188685278449664) we have missed events in two background probes:

    EventingContinualTest - testing MT Broker, 16k events propagated well, but no finished event received (no info how many events was sent)
    BrokerBackedByChannelContinualTest - testing MT Broker backed by KafkaChannel, ~15k events propagated well, ~23k request sent, but no finished event received (no info how many events was sent)

**Comment by matzew** [comment]
@cardil do we have these tests running on upstream too? Same results?

**Comment by cardil** [comment]
> do we have these tests running on upstream too? Same results?

Yes, we run them on upstream. I didn't see such problems there. If they were, those tests would be flagged as flaky, but they are not.

**Pull request closed without merging**